content/source/docs/enterprise/install/encryption-password.html.md

```diff
@@ -5,27 +5,26 @@ page_title: "Encryption Password - Install and Config - Terraform Enterprise"
 
 # Terraform Enterprise Encryption Password
 
-During the installation of Terraform Enterprise,
-a password is used to encrypt sensitive information at
-rest. The default value is auto-generated, but we
-strongly suggest you create your own password.
+When using internally-managed Vault, Terraform Enterprise requires that the operator specify a password that will be
+used to to encrypt and decrypt the internally-managed Vault unseal key and root token. This password is called the
+"encryption password". Please be sure to retain this value as it will be needed in the event of a re-installation.
 
-Be sure to retain the value, because you will need
-to use this password to restore access to the data
-in the event of a reinstall.
-
-The Encryption Password is used to protect the Vault unseal
-key and root token when the internal Vault is used.
-It allows us to store those details in PostgreSQL,
-which means that Vault is only dependent on the
-encryption password itself and details in PostgreSQL.
+The encryption password is used to protect the internally-managed Vault unseal key and root token with a password
+provided by the operator. It allows Terraform Enterprise to securely store the Vault unseal key and root token in
```

Comment:
Same question here

Reply:
Yep! Just folding lines at the 120 column instead of the 80 column like before.